build.gradle

```diff
@@ -90,52 +92,52 @@ subprojects { subproj ->
         // MobChip
         // Treasury
         maven {
-            url "https://repo.codemc.io/repository/maven-snapshots/"
+            url = "https://repo.codemc.io/repository/maven-snapshots/"
```

Comment:
Just curious if there's a reason for this change (and those like it?)

Reply:
Not necessarily, but since there were URLs in this form, I told myself that it would be consistent to do the same for everything.